composer.json

```diff
@@ -26,6 +26,7 @@
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^3.3",
+        "php-parallel-lint/php-parallel-lint": "^1.3",
```

Comment:
What do we need this for?

Reply:
For the "1️⃣ Syntax error" job.

Reply:
![kép](https://user-images.githubusercontent.com/952007/213829472-6d7b70f5-91cc-46e1-85d2-b7785ced3602.png)

Reply:
Can we use setup-php's `tools` for `parallel-lint`, since we don't run this locally using composer?

Reply:
That goes against viktor dependency policy.

Will be ready in minutes ... 🏃🏻‍♂️